scripts/docker/bash.tmp.env

```diff
@@ -0,0 +1,5 @@
+# Make temporary folder
+TEMPDIR=$(mktemp -d -t gob.XXXXXX)
+
```

Comment:
I prefer to have this a "fixed" directory so that the contents can be monitored over time, including GOB restarts and computer restarts. Files that remain in the GOB storage are normally a software error and this error should be solved.
Why not choose for the same approach as mktemp does, use TMP, TEMP if set and else use /tmp as a last resort?